Q3.js

```diff
@@ -0,0 +1,9 @@
+// Declare a variable age and initialize it with an integer, using these exact steps:
+
+// ANSWER
+let age; // declare variable
+console.log('The value of age will be: twentyFive'); // Explain the value of age
```

Comment:
The instructions might have been unclear - they were asking you to explain what you think the value of age is _before_ you initialize it.